1_Installation_Files (v1.5.5)/YOUR_ADMIN/image_handler.php

```diff
@@ -550,7 +551,8 @@ function popupWindow(url) {
         $new_button = zen_image_button('button_new_file.gif', IH_IMAGE_NEW_FILE, 'id="ih-new-button"');
 ?>
                 <tr class="dataTableRow">
-                    <td colspan="7" class="ih-right"><a href="<?php echo $new_link; ?>"><?php echo $new_button; ?></a></td>
+                    <td colspan="7" class="ih-right"><a href="<?php echo $new_link; ?>" class="btn btn-info"
+                                                        id="ih-new-button"><?= IH_IMAGE_NEW_FILE; ?></a></td>
```

Comment:
Use `<?php echo` instead of `<?=`.  Why the addition of the `id` attribute?

Put that statement on a single line.